adafruit_lsm303.py

```diff
@@ -116,6 +116,8 @@
 _LSM303ACCEL_MG_LSB        = 16704.0
 _GRAVITY_STANDARD          = 9.80665      # Earth's gravity in m/s^2
 _GAUSS_TO_MICROTESLA       = 100.0        # Gauss to micro-Tesla multiplier
+# pylint: enable=bad-whitespace
```

Comment:
Is this meant to be left in? (surprised that's not a default check)

Reply:
Yes it's meant to be there, it's needed because of the style choice to have all the registers line up, however, at the start it was disabled for the entire file which isn't consistent with the way we've been doing it, so I resolved the white space issues present in the rest of the file and enabled pylint again.

Reply:
the things we do to keep the lint monster happy